src/main/java/hudson/plugins/throttleconcurrents/ThrottleQueueTaskDispatcher.java

```diff
@@ -682,5 +689,26 @@ private int getMaxConcurrentPerNodeBasedOnMatchingLabels(
         return maxConcurrentPerNodeLabeledIfMatch;
     }
 
+    private void updatePauseAction(Task task, CauseOfBlockage cause) {
```

Comment:
How does this relate to throttling of Pipeline jobs with a throttle job property rather than a `throttle` step? Every other usage of `new PauseAction` that I can find in the `jenkinsci` organization (e.g. in Lockable Resource, SonarQube, and Pipeline: Input Step) is from a `StepExecution`. Should this be specific to a `throttle` step (see `ThrottleStep` and `ThrottleStepExecution`)?

Reply:
The pause action only applies to stages within a pipeline, so doesn't apply for pipelines using the throttle job property.